doc/tutorials/enable_ivshmem.rst

```diff
@@ -136,7 +136,6 @@ Linux-based post-launched VMs (VM1 and VM2).
         :emphasize-lines: 7
```

Comment:
When you delete a line, you'll also need to update the "emphasize-lines" arguments so the right line(s) will be highlighted. In this case it's the ``-s 6,ivshmem`` line that is now line 6:
```suggestion
        :emphasize-lines: 6
```
While we're here though, it that ``-s 6,ivshmem`` wrong because there's already a ``-s 6,virtio-hyper_dmabuf`` above?

Reply:
OK, I will check this.